File: VB-Examples/05_Paragraph/RemoveAllParagraphs.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Documents

Namespace RemoveAllParagraphs
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			'Create a new instance of the Document class.
			Dim document As New Document()

			'Load a Word document from a specified file path using relative path.
			document.LoadFromFile("..\..\..\..\..\..\Data\Template_Docx_1.docx")

			'Clear all paragraphs in each section of the document.
			For Each section As Section In document.Sections
				section.Paragraphs.Clear()
			Next section

			'Set the file name for the resulting document after removing all paragraphs.
			Dim result As String = "Result-RemoveAllParagraphs.docx"

			'Save the modified document to the specified file path in the Docx2013 format.
			document.SaveToFile(result, FileFormat.Docx2013)

			'Dispose of the document object to release any associated resources.
			document.Dispose()

			'Launch the MS Word file.
			WordDocViewer(result)
		End Sub

		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS-Examples/05_Paragraph/RemoveAllParagraphs.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;

namespace RemoveAllParagraphs
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Create a new instance of the Document class.
            Document document = new Document();

            // Load a Word document from a specified file path.
            document.LoadFromFile(@"..\..\..\..\..\..\Data\Template_Docx_1.docx");

            // Iterate through each section in the document.
            foreach (Section section in document.Sections)
            {
                // Clear all paragraphs within the section.
                section.Paragraphs.Clear();
            }

            // Specify the file name for the resulting document.
            String result = "Result-RemoveAllParagraphs.docx";

            // Save the modified document to a file with the specified file name and format (Docx2013).
            document.SaveToFile(result, FileFormat.Docx2013);

            // Clean up resources used by the document.
            document.Dispose();

            //Launch the MS Word file.
            WordDocViewer(result);
        }

        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}
